README.rst

```diff
@@ -21,6 +21,21 @@ Clients & REST API Clients code
 
 Open edX services, including LMS, should use the OAuthAPIClient class to make OAuth2 client requests and REST API calls.
 
+Usage
+~~~~~
+
+By default the ``OAuthAPIClient`` object can be used like any `requests.Session`_ object and you can follw the docs that the requests library provides.
+
+The ``OAuthAPIClient`` sessions makes some extra requests to get access tokens from the auth endpoints.  These requests have a default timeout that can be overridden by passing in a ``timeout`` parameter when instantiating the ``OAuthAPIClient`` object.
+
+.. code-block::
+    client = OAuthAPIClient('https://lms.root', 'client_id', 'client_secret', timeout=(6.1, 2.1))
```

Comment:
1. Added some clarifying docs.
2. I don't think I want to add any complexity on top of, "this is how requests does it" so I'm not gonna do that.
3. I think here is fine for now since this is a pretty small library and I want people to have some sample usage examples when they land on the main page.  If you feel strongly I can move it into the docstring for the class instead?
4. Changed.